models/claims_preprocessing/claims_enrollment/claims_enrollment__member_months.sql

```diff
@@ -0,0 +1,49 @@
+{{ config(
```

Comment:
This model has not been added to models/claims_preprocessing/claims_preprocessing_models.yml. It will be created under the user's default schema if it's not added. Was this intentional?